backend/dataall/cdkproxy/main.py

```diff
@@ -56,7 +56,12 @@ def up(response: Response):
 def check_creds(response: Response):
     logger.info('GET /awscreds')
     try:
-        sts = boto3.client('sts', region_name=os.getenv('AWS_REGION', 'eu-west-1'))
+        region = os.getenv('AWS_REGION', 'eu-west-1')
```

Comment:
To consider: Looks like we reach the same behavior if we specify `AWS_STS_REGIONAL_ENDPOINTS=regional`. boto3 sts client will make a call to a regional sts endpoint. This way we don't have to worry about forgetting to put region to sts client

Reply:
Relates to #561